docs/_docs/quickstart.md

```diff
@@ -22,7 +22,11 @@ If you already have a full [Ruby](https://www.ruby-lang.org/en/downloads/) devel
 # Now browse to http://localhost:4000
 ```
 
-If you encounter any unexpected errors during the above, please refer to the already-mentioned [requirements](/docs/installation/#requirements) page, as you might be missing development headers or other prerequisites.
+If you encounter an error in building the site, with the error message
+"'0000-00-00-welcome-to-jekyll.markdown.erb' does not have a valid date in the
```

Comment:
Please include the complete error message here for users who copy the message in a search engine and remove the `.erb` extension on the post file.

```sh
Invalid date '<%= Time.now.strftime('%Y-%m-%d %H:%M:%S %z') %>': Document '/_posts/0000-00-00-welcome-to-jekyll.markdown' does not have a valid date in the YAML front matter.
```